tools/topology/topology2/sof-ace-tplg/CMakeLists.txt

```diff
@@ -1,2 +1,59 @@
-# Empty source directory, exists only to create the corresponding
-# 'sof-ace-tplg' subdirectory in the build directory
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Array of "input-file-name;output-file-name;comma separated pre-processor variables"
+set(TPLGS
+# CAVS HDMI only topology with passthrough pipelines
+"cavs-passthrough-hdmi\;sof-ace-tplg\;sof-hda-generic-idisp\;USE_CHAIN_DMA=true"
+# CAVS HDA topology with mixer-based pipelines for HDA and passthrough pipelines for HDMI
+"cavs-passthrough-hdmi\;sof-ace-tplg\;sof-hda-generic\;HDA_CONFIG=mix,USE_CHAIN_DMA=true"
+# If the alsatplg plugins for NHLT are not available, the NHLT blobs will not be added to the
+# topologies below.
+"cavs-passthrough-hdmi\;sof-ace-tplg\;sof-hda-generic-4ch\;PLATFORM=mtl,\
+HDA_CONFIG=mix,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-mtl-hda-mix.bin"
```

Comment:
this is a weird change. You first introduce the output directory, then split the CMakeLists.txt in two so now we don't need the output directory any longer....

Reply:
fixed now